.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You shouldn't add IDE specific settings to a repository.

Reply:
Removed.